docs/concepts/tests.md

```diff
@@ -127,7 +145,7 @@ test_example_full_model:
       - num_orders: 3
 ```
 
-Leaving out the input column `item_id` means that it will be implicitly added in all input rows with a `NULL` value. Thus, we expect the corresponding output column to only contain `NULL` values, which is indeed reflected in the above test since the `item_id` column is also omitted from `query`'s rows.
+Since [omitted columns](#omitting-columns) are treated as `NULL`, this test also implicitly asserts that both the input and the output `item_id` columns are `NULL`, which is correct.
```

Comment:
Yes, that's exactly right. For example, if you had `SELECT COALESCE(item_id, 1) AS item_id FROM incremental` then that would not return a `NULL` and the test would fail. So omitting both the input and output `item_id` columns is an implicit assertion here.